src/Buffers/buffer.ts

```diff
@@ -44,7 +44,13 @@ export class Buffer {
         this._instanced = instanced;
         this._divisor = divisor || 1;
 
-        this._data = data;
+        if (data instanceof DataBuffer) {

```

Comment:
can we replace instanceOf by anthing else? isntanceof is expensive

Reply:
Do we really create so many buffers that it could be impactful?

Note that the `VertexBuffer` constructor is doing an `if (data instanceof Buffer) {`, that's why I thought I was allowed to do it in the `Buffer` constructor too :).

But I can change that in both constructors if you want.

Reply:
yeah no you are right it is just my TOC :) keep it like that no worries